operator/go.mod

```diff
@@ -1,114 +1,97 @@
 module github.com/grafana/loki/operator
 
-go 1.19
+go 1.20
```

Comment:
As discussed during our last call, we should stick to N-1 as long as we don't get production-released compilers across all our supported platforms (i.e. OpenShift).

Reply:
Right. Will correct this.